scanner/go.mod

```diff
@@ -134,6 +134,7 @@ require (
 )
 
 replace (
+	github.com/stackrox/rox => ../
```

Comment:
This is fine for this PR, but I'm noticing that every change made to `github.com/stackrox/rox` will require a `go mod tidy` both in the main module and this module, which may be annoying for devs.

I'm wondering if we should just choose the current `master` branch and then let dependabot update this weekly for us